nabo/nabo.h

```diff
@@ -365,6 +365,44 @@ namespace Nabo
 		 * 	\return an object on which to run nearest neighbour queries */
 		static NearestNeighbourSearch* createKDTreeTreeHeap(const Matrix& cloud, const Index dim = std::numeric_limits<Index>::max(), const unsigned creationOptionFlags = 0, const Parameters& additionalParameters = Parameters());
 		
+
+
+    //! Prevent creation of trees with the wrong matrix type. Currently only dynamic size matrices are supported.
```

Comment:
I'm afraid, you've got the wrong indentation white space.. This project is tab indented.